components/02-molecules/wrapped-image/yds-wrapped-image.twig

```diff
@@ -13,6 +13,8 @@
 
 {% set wrapped_image__attributes = {
   'data-component-width': wrapped_image__width|default('content'),
+  'data-component-image-style': wrapped_image__style|default('floated'),
+  'data-component-image-alignment': wrapped_image__alignment|default('left'),
```

Comment:
I think these should be component-specific data attributes. The `data-component-width` one is used by a bunch of other components. These are specific to this component. So I think they should be namespaced something like `data-wrapped-image-` in order to prevent future issues where another component might have an `-image-style` or `-image-alignment` type of data attribute.

We wouldn't want the css for this component to affect that one, or vice-versa.

Reply:
@ModulesUnraveled I have renamed them. Great point, thank you!